sprokit/processes/core/frame_list_process.cxx

```diff
@@ -78,6 +78,10 @@ create_config_trait( frame_time, double, "0.03333333", "Inter frame time in seco
                      "timestamps for sequential frames. This can be used to simulate a frame rate in a "
                      "video stream application.");
 
+create_config_trait( no_path_in_name, bool, "true",
```

Comment:
It is usually better to state these options in the positive, such "full_path_file_name" and then set to false. Remember the vidtk process disable option.